docs/source/index.rst

```diff
@@ -1,21 +1,91 @@
-Welcome to probnum's documentation!
-======================================================
+===================================
+Probabilistic Numerics in Python
+===================================
 
+|Travis Status| |Coverage Status| 
+
+
+`Probabilistic numerics (PN) <http://probabilistic-numerics.org/>`_ (PN) interprets classic numerical routines as 
+*inference procedures* by taking a probabilistic viewpoint. This allows principled treatment of *uncertainty arising 
+from finite computational resources*. The vision of probabilistic numerics is to provide well-calibrated probability 
+measures over the output of a numerical routine, which then can be propagated along the chain of computation.
+
+This repository aims to implement methods from PN in Python 3 and to provide a common interface for them. This is
+currently a work in progress, therefore interfaces are subject to change.
+
+To get started install ProbNum from the Python Package Index via
+
+.. code-block:: shell
+   
+   pip install probnum
```

Comment:
Is this valid already? Or will this be valid as soon as we push to master?

Reply:
Not actually valid yet. My goal was to resolve this asap after the new docs. This does create a period of the documentation being inaccurate though.

Reply:
Thanks, good to know!